README.md

```diff
@@ -2,16 +2,19 @@
 
 # Project Name
```

Comment:
- Please kindly add the meaningful name of the project and a short description of the project below the project name, to allow people who will see this repo to have an idea about your project :slightly_smiling_face:
- Kindly also add a screenshot for the project :framed_picture: